src/apps/embedding.C

```diff
@@ -179,7 +179,7 @@ int main(int argc, char ** argv)
             }
           else
             {
-              int oldnumerator = std::round(shape*denominator);
```

Comment:
This is wild.  Does boost 128 convert to int explicitly but not implicitly?

Reply:
Indeed, https://godbolt.org/z/ve9zEaqza.